### PR TITLE
Updated README for Texture benchmark

### PR DESCRIPTION
**Pull request by wsu-nyt**
This PR updates README by adding Texture related information. I've requested edit access on [Google Spreadsheet Document ](https://docs.google.com/spreadsheets/d/1sUNdGWBM-d_W13yC7VcfkRXC3owCVsnIublnfW-4xn4/edit#gid=1032991425), so that we can update XLSX document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,10 @@ Layout your views without Auto Layout constraints, in a much more swifty way.
 Fast Swift UIViews layouting without auto layout. No magic, pure code, full control and blazing fast. Concise syntax, intuitive, readable & chainable.  
 [PinLayout benchmark's source code](https://github.com/layoutBox/LayoutFrameworkBenchmark/blob/master/LayoutFrameworkBenchmark/Benchmarks/PinLayout/FeedItemPinLayoutView.swift)
 
+* [**Texture**](https://github.com/TextureGroup/Texture)  
+Optimize your app by making user interfaces thread safe, which means that you will be able to shift all expensive views into background threads.  
+[Texture benchmark's source code](https://github.com/wsu-nyt/LayoutFrameworkBenchmark/blob/master/LayoutFrameworkBenchmark/FeedItemTextureNode.swift)
```
Comment (ZevEisenberg) [inline README.md:81]: this should link to the main repo, not your fork

**Comment by ZevEisenberg** [comment]
@wsu-nyt if you use ⌘S in the iPhone Simulator, it will save a screenshot to your desktop without the device frame, so you'll just have the app. This is useful in general, but it would be especially good here so it matches the other screenshots in the repo :)

File: README.md [hunk 2]
```diff
@@ -98,6 +102,7 @@ Here are the benchmark rendering results to compare visual results:
 * [Manual layout rendering result](docs_markdown/benchmark_result_ManualLayout.png)
 * [NKFrameLayoutKit rendering result](docs_markdown/benchmark_result_NKFrameLayoutKit.png)
 * [PinLayout rendering result](docs_markdown/benchmark_result_PinLayout.png)
+* [Texture rendering result](docs_markdown/benchmark_result_Texture.png)
```
Comment (lucdion) [inline README.md:105]: In the rendering results, there are some issues:

1. The top section should look like this 
image
- Wrong vertical alignment
- Image size
- Orange background color

2. Missing some bottom padding. It should look like this
image

3. Missing some bottom padding. It should look like this
image

4. The label should be vertically aligned. It should look like this
image

Comment (wsu-nyt) [reply README.md:105]: Rendering results issues should be fixed now @lucdion .

**Review by lucdion — APPROVED** [review]
👍

**Pull request merged**